### DISCUSSION TITLE
How does everyone use Axiom?

### DISCUSSION THREAD

**Discussion by ResistanceIsUseless** [pry0cc/axiom#432] [Q&A]
I would be curious to know how everyone uses Axiom. Do you mainly use it to spin up fleets to use axiom-scan? Or do you spin up Axiom and ssh into it for a quick way to spin up a VM that has what you need per-configured? Maybe it's something else?

**Comment by ResistanceIsUseless** [comment]
Personally I mainly use axiom as a way to spin up a VM to test out ideas and test out tools. My objective being to build security automation tools. So it would be nice if there was a way to backup and restore specific personal files and configurations.

**Comment by mrtnrdl** [comment]
I'm using axiom most of the time to automate my inital recon of targets. So I usually spawn a fleet of 5-20 instances and let reconftw run with them. 

sometimes i also use it to catch my reverse shells, if the pentest goes _really_ well and we get to that stage.

**Reply by parvbarot** [reply]
Whenever I use Reconftw with Axiom, I couldn't find any findings with Nuclei
